src/drivers/github.js

```diff
@@ -258,7 +258,8 @@ class Github {
       name,
       labels: labels.map(({ name }) => name),
       online: status === 'online',
-      busy
+      busy,
+      status
```

Comment:
Why is this needed?

Reply:
We were doing it before, we need to check the status of completion

Reply:
Where are we using the value of this property in the code? 🤔

Reply:
And, what's the relationship between the `status` returned by this function and the completion of a job?

Reply:
```js
.filter((job) => job.status === 'completed')
```

Reply:
Its true. I added it to check the runner status with the first solution

Reply:
That `filter` is taking the output from `cml.pipelineJobs` and not `cml.runners` unless I missed something. 🤔

Reply:
Anyway is a nice field

Reply:
> That filter is taking the output from cml.pipelineJobs and not cml.runners unless I missed something. 🤔

Yes,I remembered the first approach that were runners indeed

Reply:
I'm no expert in measuring the niceness of fields, but it has been superseded by `online` and `busy` fields, that offer the same values across all the supported platforms.

Reply:
It might not be enough actually. And i would say that I would deprecate online and busy in favour of status which is richer and something that we are exposing for the viewer

Reply:
The `status` field doesn't offer the same values across platforms, while `online` and `busy` do. Nevertheless, we should consider this separately; the `status` field is not necessary for this pull request.

Reply:
```suggestion
```